VERSION

```diff
@@ -1 +1 @@
-1.7.2
+1.7.3
```

Comment:
Version 1.7.2 was not released yet, so this change is potentially unnecessary, see #22

Reply:
Reverted